cg/meta/demultiplex/wipe_demultiplex_api.py

```diff
@@ -0,0 +1,227 @@
+import logging
+import shutil
+
+from glob import glob
+from pathlib import Path
+from typing import Iterable, List, Optional
+
+from cg.apps.housekeeper.hk import HousekeeperAPI
+from cg.apps.cgstats.stats import StatsAPI
+from cg.exc import WipeDemuxError
+from cg.models.cg_config import CGConfig
+from cg.store import Store
+from cg.store.models import Sample, Flowcell
+from housekeeper.store.models import File
+
+log = logging.getLogger(__name__)
+
+
+class WipeDemuxAPI:
```

Comment:
It's a bit unclear to me what Wipe actually means, could we call it delete if it is what it is actually about? 🙂

Reply:
okay! 👍